test/makecert.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# # if you need a new self-signed cert to test https
+openssl genrsa -out "tls.key" 2048
+openssl req -new -key "tls.key" -out "tls.csr" -subj '/C=FI/ST=Uusimaa/L=Espoo/O=CSC/CN=localhost'
+openssl x509 -req -days 365 -in "tls.csr" -signkey "tls.key" -out "tls.crt"
```

Comment:
```suggestion
openssl x509 -req -days 7300 -in "tls.csr" -signkey "tls.key" -out "tls.crt"
```
to keep consistent with: https://github.com/neicnordic/sda-pipeline/blob/master/dev_utils/make_certs.sh